services/address-lookup/README.md

```diff
@@ -0,0 +1,97 @@
+# address-lookup
+
+A service to retrieve UK postcodes by the postcode.
+
+The service consumes the OS Apis (https://apidocs.os.uk/), specifically the DPA dataset (https://apidocs.os.uk/docs/os-places-dpa-output).
+
+## Installation
+
+    npm i @planning-inspectorate/address-lookup
+
+And add the API key in  .env of your project:
+
+    OS_API_KEY = {insert key}
```

Comment:
This is a little one, but I might suggest using `OS_PLACES_API_KEY`, just because it's a little more descriptive?